environments.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
```

Comment:
It almost looks like both files could be combined into a `config.json`. Having 2 config files plus 2 schemas in the root folder seems overly modularised.